README.md

```diff
@@ -3,38 +3,37 @@
 [![Build Status](https://img.shields.io/travis/JetBrains/youtrack-scripting/master.svg?style=flat-square)](https://travis-ci.org/JetBrains/youtrack-scripting)
 [![official JetBrains project](http://jb.gg/badges/official-flat-square.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 
-The **@jetbrains/youtrack-scripting** package contains utility scripts that help manage YouTrack workflows when you work in an external code editor. This lets you write and update workflows for YouTrack in JavaScript in your preferred development environment.
+The **@jetbrains/youtrack-scripting** package contains utilities that help you manage YouTrack workflows when you work in an external code editor. This lets you write and update workflows for YouTrack in JavaScript in your preferred development environment.
 
 ## Quick Start
 
 To work with the scripting package, you need to install and run [Node.js](https://nodejs.org/en/). This also installs the npm package manager that lets you work with the scripting package in your projects.
-Next, install the **@jetbrains/youtrack-scripting** package in your development environment. The installation process varies based on the tool that you use to write your code. If you’re working with an IDE from JetBrains, follow the instructions provided in the documentation:
-* [IntelliJ IDEA](https://www.jetbrains.com/help/idea/node-js.html)
-* [PhpStorm](https://www.jetbrains.com/help/phpstorm/npm.html)
-* [WebStorm](https://www.jetbrains.com/help/webstorm/2017.2/npm.html)
-
-The easiest way to get started is to install the package globally with the following command:
+Next, install the **@jetbrains/youtrack-scripting** package in your development environment. The easiest way to get started is to install the package globally with the following command:
 
 ```npm install -g @jetbrains/youtrack-scripting```
 
+If you prefer to install packages as dependencies in your development environment, enter:
+
+```npm install --save-dev @jetbrains/youtrack-scripting```
+
 ## Utility Scripts
 
 The package includes scripts that let you synchronize local changes with your YouTrack installation. The following commands are available:
 
 ### List
 
-`youtrack-workflow list --host [--token]`
+`youtrack-workflow list --host --token`
 
 This command lists all of the workflows that are available in your YouTrack installation. To use this command, specify the following parameters:
 
 Parameter | Description
 --- | :---
-host | The domain name of your YouTrack installation.
+host | The base URL of your YouTrack installation. For an InCloud instance, include the trailing `/youtrack`.
 token | A permanent token that grants access to the YouTrack service. You can generate your own permanent tokens to authenticate with YouTrack on the **Authentication** tab of your Hub profile.
 
 ### Download
 
-`youtrack-workflow download <workflowName> [--output]`
+`youtrack-workflow download -- <workflowName> --host --token [--output]`
```

Comment:
In this situation ' -- ' is not required before the workflow name. It is required when you add your one commands to you `scripts` section in package.json. In this case it separates the parameter of `npm run` (which is, say, `upload-prod`) and the command itself (which is the name of the workflow).